### ISSUE TITLE
Add support for checking/generating API of testFixtures

### ISSUE THREAD

**Issue by remcomokveld** [Kotlin/binary-compatibility-validator#117]
Kotlin JVM libraries support defining test fixtures using the `java-test-fixtures` (https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures) gradle plugin. These test fixtures can be published and should thus be considered part of the public API.

It would be nice if the binary-compatbility-validator would therefore generate a separate `*.api` file with the API of the test fixtures

**Comment by aSemy** [comment]
I've implemented test-fixtures API generation in my fork of BCV - if you'd like to give it a go then I'd appreciate the feedback 

https://github.com/adamko-dev/kotlin-binary-compatibility-validator-mu

**Comment by tso** [comment]
I believe https://github.com/Kotlin/binary-compatibility-validator/pull/116 would solve this. Is that correct @qwwdfsad and could you do a release if true? 🙏

**Comment by tso** [comment]
Update: Can confirm that this is solved with version 0.13.1 by configuring like so:

```kotlin
apiValidation {
  additionalSourceSets.addAll(listOf("testFixtures"))
}
```
